docs/variants/novacustom_nv4x_adl/hardware-matrix.md

```diff
@@ -5,6 +5,16 @@
 This document describes the hardware configurations used for validation of the
 coreboot port on the NovaCustom NV4X laptop.
 
+## Ports specification
```

Comment:
@Pokisiekk Instead of adding explanations on the graphic, couldn't we gather them in a table (e.g., with an additional column)?

The proposed approach is problematic in that the graphics will have to be updated with each change.